packages/blank-theme/src/types/company-filter.ts

```diff
@@ -0,0 +1,181 @@
+export type ICompanyFilterData = {
+   id: number
+   __component: string
+   title: string
+   description: string | null
+   search: string
+   industry: string[]
+   company_size: string[]
+   revenue: string[]
+   showFilter: boolean
+   average_salary: string[]
+   button?: {
+      id: number
+      label: string
+      link: string | null
+      type: string | null
+      target: string | null
+      icon: {
+         data: any | null
+      }
+   }
+   companies: {
+      data: ISingleCompany[]
+   }
+}
+
+export type ISingleCompany = {
```

Comment:
Need to review

Reply:
@zelal-dev vai, I didn't work with this `blank theme`. Fixing this issue will be a different task.